LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -167,6 +160,33 @@ extension TherapySettingsView {
         }
     }
     
+    private var suspendThreshold: some View {
+        section(for: .suspendThreshold) {
+            Group {
+                if self.glucoseUnit != nil {
```

Comment:
[just thinking aloud here, no action required]

I wonder if we can get to where `glucoseUnit` is non-optional here, too.. I know this is tough, since most `glucoseUnit` properties across the codebase are optional. Might be a tougher refactoring, though it makes sense semantically that a unit preference must exist.

Reply:
I agree...it doesn't "smell" great to be checking nil glucose unit here.  I'd like to think about it for a future improvement... (post this PR ;) )